src/pages/en/guides/images.md

```diff
@@ -27,10 +28,11 @@ import rocket from '../images/rocket.svg';
 <img src={rocket} alt="A rocketship in space."/>
 ```
 
-### In `.md` files
+### In Markdown files
 
 You can use standard Markdown `![]()` syntax or standard HTML `<img>` tags in your `.md` files for images located in your `public/` folder, or remote images on another server.
 
+If you cannot keep your images in `public/`, we recommend using the Astro MDX integration to allow you to import and include components in Markdown.
```

Comment:
My suggestion here could probably do with some further massaging, but it might be worth being specific that the integration doesn’t actually let you do extra stuff **IN** Markdown — you still need to switch to another format.

```suggestion
If you cannot keep your images in `public/`, we recommend using the `.mdx` file format, which allows you to combine imported components with Markdown-like syntax. Use the [MDX integration](/en/guides/integrations-guide/mdx/) to add support for MDX to Astro.
```

(I could also see an argument for moving this line _after_ the code sample? But I do like the consistent pattern of: heading, text, code sample.)